carmm/build/neb/bond_length_scan.py

```diff
@@ -81,12 +86,12 @@ def take_second(n):
 
     for i in range(1, n_steps+1):
         # operate on a deepcopy for intended functionality
-        atoms = copy.deepcopy(atoms)
+        atoms = atoms.copy()
```

Comment:
This now that you working on the original structure, not a copy - can I just confirm this is intended?

Reply:
Yes, in previous versions atoms.copy() was providing a shallow copy and now it is a completely new instance, so using `copy` library can be avoided